data/build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '3.1.2'
+	id 'io.spring.dependency-management' version '1.1.2'
+}
+
+group = 'com.example'
+version = '0.0.1-SNAPSHOT'
+
+java {
+	sourceCompatibility = '17'
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-data-jdbc'
+	testImplementation 'org.springframework.boot:spring-boot-starter-test'
```

Comment:
I would recommend organize `dependencies` section in some way. E.g. annotationProcessor and compileOnly go first, then all `implementation` and `runtimeOnly' and all `testImplementation` go last.

Another good idea is to separate all that logical parts by extra empty line